### ISSUE TITLE
does not read from tcp client to serial

### ISSUE THREAD

**Issue by harryberlin** [maks/UartBridge#9]
I'm very new in andoid app programming.
got it compiled in AS 4.0.1.
Did some tests, but i can't send from tcp to serial.
looks like the tcp server don't read.

How to get it working, read the tcp input and write to uart?
Is there missing a method to read and/or write to uart?

**Comment by maks** [comment]
Yes this app was meant for a very limited purpose of just relaying data coming in from a uart (on a usb port) to an app that can only have a TCP port as input. So I never implemented sending any data back via TCP out to the uart, so would need to implement that if you need that functionality.

**Comment by DimitarYakov** [comment]
Same here, it would be nice if you can make that option.

**Comment by maks** [comment]
I'm afraid Im not likely to get to adding this feature anytime soon due to other commitments, but I'd be happy if someone wanted to contribute a PR for this.

**Comment by DimitarYakov** [comment]
Sadly i`m new in adroid and dont have that much experience with java (mostly data processing projects), don't promise anything, but i`ll try my best, that kind a tool would be useful in some cases.
